package.json

```diff
@@ -103,10 +103,10 @@
     "lint-staged": "^9.2.1",
     "prettier": "^1.18.2",
     "prop-types": "^15.7.2",
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6",
+    "react": "^16.9.0",
```

Comment:
We need to increase the versions in the packages to match this too

Reply:
Thanks, forgot about peer deps

Reply:
Done, let me know if it's good to do a patch update on.